.github/codecov.yml

```diff
@@ -0,0 +1,26 @@
+coverage:
+  status:
+    project:
+      default:
+        target: 50%
+      client: 
+        target: 50%
+        threshold: 1%
+        flags:
+          - client
+      server:
+        target: 100%
+        threshold: 1%
+        flags:
+          - server
+
+comment:
+  layout: "diff, flags, files"
+
+flags: 
```

Comment:
We should be able to remove this flags here. And instead let's rename the existing flags to "client" and "server" here: https://github.com/paypal/paypal-sdk-client/blob/main/.github/workflows/main.yml#L32-L42